### PR TITLE
Add Home Assistant template

### PR DESCRIPTION
**Pull request by nebuto**

### CODE REVIEW

**Comment by anthonyray** [comment]
No template for the hass `configuration.yml`?

**Comment by nebuto** [comment]
It isn't supported by `snips-template`, I will do it with SAM. But it is a good idea indeed.

File: templates/homeassistant/spec.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name":"homeassistant",
+  "language":"PYTHON",
+  "prelude": "\"\"\"\nsnips_skill_slot_value = data.get('snips_skill_slot_value')\nservice_data = {'entity_id': 'snips_skill_slot_value' }\nhass.services.call('hass_component', 'hass_component_method', service_data, False)\"\"\"",
```
Comment (rby) [inline templates/homeassistant/spec.json]: `snips_skill_slot_value` is confusing. Skills are a set of intent and associated actions.
maybe `<intent_slot_name>` is more appropriate so the user at least knows that he has to change this

Comment (nebuto) [reply templates/homeassistant/spec.json]: Yep you're right, it's a bit confusing, updated.

**Pull request merged**